client/components/gsuite/docs/new-user-list.tsx

```diff
@@ -85,17 +85,18 @@ const GSuiteNewUserListExample = (): React.FunctionComponent => {
 			</GSuiteNewUserList>
 			<hr />
 			<FormLabel key="mulitple-domains">
-				<FormToggle checked={ useMultipleDomains } onChange={ toggleUseMultipleDomains }>
-					Use multiple domains
-				</FormToggle>
+				<ToggleControl
+					checked={ useMultipleDomains }
+					onChange={ toggleUseMultipleDomains }
+					label="Use multiple domains"
+				/>
 			</FormLabel>
 			<FormLabel key="extra-validation">
-				<FormToggle
+				<ToggleControl
 					checked={ useExtraValidation }
 					onChange={ () => setUseExtraValidation( ! useExtraValidation ) }
-				>
-					Use extra validation ( no a's in name )
-				</FormToggle>
+					label="Use extra validation ( no a's in name )"
```

Comment:
I wonder if these should be translated?

Reply:
I don't think so. This is in the component documentation, so we're intentionally not translating those.